tools/README.md

```diff
@@ -5,34 +5,64 @@ This tool allows ballerina message broker developers to get an idea about the pe
 - Connection Load  - The number of message producers , or the number of concurrent connections a system can support.
 - Message throughput - The number of messages or message bytes that can be pumped through a messaging system per second.
 
-At the moment the tool allows developers to publish messages to queues and test the performance and get a report of the result.
+At the moment the tool allows developers to publish/consume messages to/from queues and topics and test the performance and get a report of the result.
```

Comment:
Let's not say "at the moment". Instead say start with "The tool ...."

Reply:
Done